0.download_data/download_jump_data.py

```diff
@@ -0,0 +1,51 @@
+r"""°°°
```

Comment:
When you `nbconvert` these files, please put them in a subfolder so as not to confuse someone looking at your repository.

I typically put them in a folder called `nbconverted`.